Content/Items/Gear/Weapons/Grimoire/GrimoireItem.cs

```diff
@@ -0,0 +1,60 @@
+﻿using PathOfTerraria.Content.GUI.GrimoireSelection;
+using PathOfTerraria.Content.Projectiles.Summoner;
+using PathOfTerraria.Core;
+using PathOfTerraria.Core.Loaders.UILoading;
+using PathOfTerraria.Core.Systems.ModPlayers;
+using Terraria.ID;
+using Terraria.Localization;
+
+namespace PathOfTerraria.Content.Items.Gear.Weapons.Grimoire;
+
+internal class GrimoireItem : Gear
+{
+	public override string Texture => $"{PathOfTerraria.ModName}/Assets/Items/Gear/Weapons/Grimoire/{GetType().Name}";
+	public override float DropChance => 1f;
```

Comment:
Shouldn't this be 0f?

Reply:
I think I had this set to 1 when we were talking about adding affixes directly to the Grimoire. Will remove!